packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```
@@ -18,6 +18,22 @@
     }
 }
 
+.dc-input .dc-input {
```

I think we don't need parent class 
```suggestion
.dc-input {
```

---

it is needed to avoid using `!important`. Parent node has `dc-input` class

---

Can we give this field a `className`?